pfcpiface/parse-far_test.go

```diff
@@ -0,0 +1,48 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2022-present Open Networking Foundation
+
+package main
+
+import (
+	"net"
+	"testing"
+
+	"github.com/omec-project/upf-epc/test/integration"
+)
+
+func TestFARParsing(t *testing.T) {
+	f := far{}
+
+	op := create
+
+	var validFARID uint32 = 1
+
+	var validFARAction uint8 = ActionForward
+
+	validFAR := integration.NewUplinkFAR(integration.IEMethod(op), validFARID, validFARAction)
+	// using invalid action (0)
+	invalidFAR := integration.NewDownlinkFAR(integration.IEMethod(op), 2, 0, 100, "10.0.10.1")
+
+	mockUpf := &upf{
+		accessIP: net.ParseIP("192.168.0.1"),
+		coreIP:   net.ParseIP("10.0.0.1"),
+	}
+
+	err := f.parseFAR(validFAR, 100, mockUpf, op)
+	if err != nil {
```

Comment:
We should make use of `require` or `assert` package to verify.

Reply:
please apply it to all checks below

Reply:
Done